docs/all-proxy-resources-should-have-delete.md

```diff
@@ -12,22 +12,299 @@ ARM OpenAPI(swagger) specs
 
 - RPC-Delete-V1-05
 
-## Output Message
+## Description
 
-The resource {resourceName} does not have a corresponding delete operation.
+All proxy resources that have a put operation defined should also support a delete operation. This makes the APIs intuitive for customers as they would expect every resource that they have the ability to create to also be deletable. There are rare sceanrios where a customer is allowed to create and replace a resource but never be able to delete them. In such cases it is ok to not implement a delete but such cases are very rare and should be avoided. 
 
-## Description
+## How to fix
+
+Add the delete operation for the proxy resource.
```

Comment:
an we add "If PUT exists" to the end as DEL isnt mandatory if PUT isnt defined

Reply:
The assumption is that the PUT doesnt exist as thats when the rule flags an error but I can make it explicit.